securejarhandler/src/main/java/cpw/mods/cl/ModuleClassLoader.java

```diff
@@ -117,11 +115,15 @@ public ModuleClassLoader(final String name, final Configuration configuration, f
             // Loading a class requires its module to be part of resolvedRoots
             // If it's not, we delegate loading to its module's classloader
             if (!this.resolvedRoots.containsKey(k.name())) {
-                return parentLayers.stream()
-                        .filter(l -> l.configuration() == k.configuration())
-                        .flatMap(layer -> Optional.ofNullable(layer.findLoader(k.name())).stream())
-                        .findFirst()
-                        .orElse(ClassLoader.getPlatformClassLoader());
+                for (var parentLayer : parentLayers) {
+                    if (parentLayer.configuration() == k.configuration()) {
+                        var loader = parentLayer.findLoader(k.name());
+                        if (loader != null) {
+                            return loader;
+                        }
+                    }
+                }
+                return fallbackClassLoader;
```

Comment:
This was changed from `ClassLoader.getPlatformClassLoader()`. This is probably fine, but please double-check.